locales/en-US.json

```diff
@@ -449,5 +450,9 @@
     "footer_source_code": "Source code",
     "footer_original_source_code": "Original source code",
     "footer_modfied_source_code": "Modified Source code",
-    "adminprefs_modified_source_code_url_label": "URL to modified source code repository"
+    "adminprefs_modified_source_code_url_label": "URL to modified source code repository",
+    "Embed": "Embed",
+    "Embed Link": "Embed Link",
+    "Subtitles - `x` (.vtt)": "Subtitles - `x` (.vtt)",
```

Comment:
While you're at it, could you normalize these keys?

Reply:
yup. Will do!